src/core/mainloop.rkt

```diff
@@ -290,33 +289,6 @@
        (combine-alts opt ctx))]
     [else (list (argmin score-alt alts))]))
 
-(define (final-simplify! alts)
-  (cond
-    [(flag-set? 'reduce 'simplify)
-     (timeline-event! 'simplify)
-
-     ; egg schedule (only FP rewrites plus simplify rewrites for if statements)
-     (define rules (append (platform-simplify-rules) (*simplify-rules*)))
```

Comment:
I believe `platform-simplify-rules` can also be removed

Reply:
There's probably more functions around `platform-simplify-rules` that can be removed as well